app/Jobs/CreateRockTheVotePostInRogue.php

```diff
@@ -356,35 +364,18 @@ private function updateStatus($currentStatus, $newStatus)
         return $indexOfCurrentStatus < $indexOfNewStatus ? $newStatus : null;
     }
 
-
-    /*
-     * Translate "Opt-in to Partner SMS/robocall" from Rock the Vote CSV to a Northstar sms_status
-     *
-     * @param array $sms_status
-     * @return string
-    */
-    private function transformSmsStatus($sms_status)
-    {
-        if ($sms_status === 'Yes') {
-            return 'active';
-        }
-
-        // @TODO: do we want this to be 'pending' or some other status? we talked about this recently referring to something else
-        return 'stop';
-    }
-
     /*
      * Translate to Northstar status and update Northstar user (Northstar takes care of the hierarchy)
      *
-     * @param Object $user
+     * @param string $userId
      * @param string $statusToSend
     */
-    private function updateNorthstarStatus($user, $statusToSend)
+    private function updateNorthstarStatus($userId, $statusToSend)
```

Comment:
Same thought here: passing a user instance (if we added a `NorthstarUser $user` type hint) would avoid the chance of sneaky bugs if we accidentally passed the wrong kind of ID to this function.

Reply:
I changed this up to avoid fetching a user when the csv already has a `northstar_id` column when completing that TODO in https://github.com/DoSomething/chompy/pull/58/commits/3c09ebded9850571da151d0006a3cdde52eda7aa. It feels odd to wrap a dummy Northstar user object/array around the id.. but also to fetch the user when we already have a NS ID? (Unless we want super sanity checking to ensure the file NS ID is correct)

Reply:
I think it's a good idea to sanity-check the ID exists in these cases. (We've had issues with bad data being provided to Chompy before, so an invalid Northstar ID doesn't seem out of the question).